bigwig_loader/collection.py

```diff
@@ -75,6 +73,27 @@ def run_indexing(self) -> None:
     def __len__(self) -> int:
         return len(self.bigwigs)
 
+    def reset_gpu(self) -> None:
```

Comment:
Is this ever called on the collection level?

It seems you have to repeat the method to have to delegate from the top levels down.

Reply:
Yes. Because a BigWigDataset object can take a BigWigCollection object as one of the init args. This can be handy when you want to create train, val and test datasets, all based on the same collection of bigwig files, but with different "regions of interest" from which the samples are taken: 

https://github.com/pfizer-opensource/bigwig-loader/blob/d84a233a6214162c3e7178684f43d8a198fb88d0/bigwig_loader/dataset.py#L61C1-L62C1